en/micro-integrator/docs/administer-and-observe/using-the-command-line-interface.md

```diff
@@ -1,486 +1,776 @@
 # Micro Integrator CLI
 
-The Micro Integrator CLI allows you to monitor the synapse artifacts (deployed in a specified Micro Integrator server) from the command line. The CLI is an alternative to the [management dashboard](../../administer-and-observe/working-with-monitoring-dashboard), which provides a graphical view of the deployed artifacts.
+The Micro Integrator CLI allows you to monitor the synapse artifacts (deployed in a specified Micro Integrator server) from the command line. The dashboard (as well as the [dashboard](../../administer-and-observe/working-with-monitoring-dashboard)) communicates with the management API of WSO2 Micro Integrator to function.
 
-The dashboard as well as the CLI communicates with the management API of WSO2 Micro Integrator to function. Therefore, be sure to [enable the Management API](#enable-the-management-api) in the server before using the dashboard or the CLI.
+## Installing the CLI
 
-## Enable the Management API
+1.  To download the CLI, go to [**WSO2 Micro Integrator** website](https://wso2.com/integration/micro-integrator/#) -> **Download** -> **Other Resources**, and click **CLI Tooling**.
+2.  If you are using a UNIX-based operating system (Linux, Solaris, and Mac OS X), be sure to set the `MI_CLI_HOME/bin` folder path as the PATH:
 
-To use the CLI tool, you need to enable the management API when you
-start your WSO2 Micro Integrator instance. Pass the following system property:
+    ```bash
+    export PATH=/path/to/mi/cli/directory/bin:$PATH
+    ```
 
-```bash
--DenableManagementApi
-```
+## Using the CLI
+
+1. **Start the CLI**
 
--   When you run the Micro Integrator on Docker, start your Docker
-    container by passing the `enableManagementApi` system property:
+    Execute the following command to start the CLI:
 
     ```bash
-    docker run -p 8290:8290 -p 9164:9164 -e JAVA_OPTS="-DenableManagementApi=true" <Docker_Image_Name>
+    ./mi
```

Comment:
if the command starts with **./** , we need to run from bin directory. Since we have asked to add the CLI to path in step 2, we can just say **mi**